PluginJira/API/Utility/EndpointHelperEndpoints/ApplicationRolesEndpoint.cs

```diff
@@ -53,8 +53,7 @@ public override async Task<Count> GetCountOfRecords(IApiClientFactory factory, S
 
                 return new Count
                 {
-                    Kind = Count.Types.Kind.Exact,
-                    Value = (int) recordsList.Count()
+                    Kind = Count.Types.Kind.Unavailable
```

Comment:
If you return unavailable you should not perform any other actions in the count method